fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java

```diff
@@ -84,6 +85,20 @@ public File getBinaryDirectory() {
     }
 
     /**
+     * Sets the Binary filename extension.
+     */
+    public void setBinaryExtension(final String extension) {
```

Comment:
[WARNING] Javadoc Warnings
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo-import-export/fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java:96: warning: no @param for extension
[WARNING] public void setBinaryExtension(final String extension) {
[WARNING] ^
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo-import-export/fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java:103: warning: no @return
[WARNING] public String getBinaryExtension() {
[WARNING] ^